Assets/DLC/Expansion/Gameplay/XML/Text/pt_BR/CIV5GameTextInfos_Medieval_Scenario.XML

```diff
@@ -380,10 +380,10 @@
 		</Row>
 		<!-- Unique units and buildings -->
 		<Row Tag="TXT_KEY_MEDIEVAL_SCENARIO_UNIT_HELP_MUSLIM_CAVALRY">
-			<Text>Powerful Medieval Mounted Unit, weak to Pikemen. Only the Ayyubid may build it. This Unit is not penalized when attacking Cities, unlike the Knight which it replaces.</Text>
+			<Text>Poderosa Unidade Montada Medieval, fraca contra {TXT_KEY_UNIT_PIKEMAN[2]}. Apenas os Aiúbidas podem treiná-la. Essa Unidade não é penalizada quando ataca Cidades, diferente do {TXT_KEY_UNIT_KNIGHT} que ela substitui.</Text>
```

Comment:
Acho que "Unidade Medieval Montada Poderosa" ou "Poderosa Unidade Medieval Montada" ficaria melhor.

Reply:
Não concordo, acho que o adjetivo mais importante aqui é Montada, e portanto deve ficar mais próximo do substantivo, assim como no original em inglês.

Reply:
O que acham de "Poderosa Unidade Medieval Montada"?
A unidade é medieval, e o montada é um adjetivo, como um status.

Reply:
Como eu disse acima, tanto Montada como Medieval são adjetivos, e preferi manter Montada mais perto do substantivo para manter a padronização do original em inglês, aqui e em vários outros lugares onde "Unidade Montada" recebe este e outros adjetivos.  Esse termo aparece assim mais de uma vez em 3 arquivos (CIV5GameTextInfos_Units.xml, CIV5GameTextInfos_Medieval_Scenario.xml, CIV5GameTextInfos_1066_Scenario.xml), já "unidade medieval montada" não aparece nenhuma vez.  Se realmente for a tradução preferida, sugiro fazer um PR separado padronizando isso. :)